### PR TITLE
Move back to bellsoft image for arm

### PR DESCRIPTION
**Pull request by kmcnellis**
### Description

OpenJDK doesn't successfully compile for arm (it randomly times out).  Move back to the bellsoft image re: https://github.com/adoptium/containers/issues/158#issuecomment-1046890267
## Release notes:

n/a

### Checklist

- [ ] Added the correct label: < feature | bug | dependencies | infrastructure | ignore-for-release >
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### Issue(s)

Fixes #<issue_number>; Fixes #<issue_number>; Fixes #<issue_number>...

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # syntax=docker/dockerfile:1
-FROM alpine
+
+# The eclipse-temurin image and the standard openJDK11 fails to run on M1 Macs because it is incompatible with ARM architecture. This
+# workaround uses an aarch64 (arm64) image instead when an optional platform argument is set to arm64.
+# Docker's BuildKit skips unused stages so the image for the platform that isn't used will not be built.
+
+FROM eclipse-temurin:11.0.15_10-jdk-alpine as amd64
+FROM bellsoft/liberica-openjdk-alpine:11.0.16-8 as arm64
```
Comment (gwendolyngoetz) [inline Dockerfile:8]: I wonder if we should keep temurin and bellsoft set at the same version number. My thought is that for devs we've seen minor issues when people (ok, often me) have been on different versions of docker-compose. Having ARM Macs vs Intel Macs/PCs with different version could give us troubleshooting headaches.

**Review by azizova-leyla — APPROVED** [review]
It works! Thank you for digging into this. Please address Gwen's comment, but lgtm otherwise.

File: Dockerfile (hunk 1 above)
Comment (kmcnellis) [reply Dockerfile:8]: We could pin them to the same version, but we'll have to make sure that the version is published to both places before updating it

Comment (kmcnellis) [reply Dockerfile]: Correction: no.  They don't publish exactly the same versions

Comment (gwendolyngoetz) [reply Dockerfile]: OK. Thanks

**Pull request merged**